ADAL/src/cache/ADAuthenticationContext+TokenCaching.h

```diff
@@ -31,11 +31,13 @@
 //the item to be stored.
 - (void)updateCacheToResult:(ADAuthenticationResult*)result
                   cacheItem:(ADTokenCacheItem*)cacheItem
-           withRefreshToken:(NSString*)refreshToken;
+           withRefreshToken:(NSString*)refreshToken
+       requestCorrelationId:(NSUUID*)requestCorrelationId;
```

Comment:
Rejected refresh token is removed within this function, so request correlation ID should be passed in in order to do the tombstone action.